video_xblock/static/html/wistiavideo.html

```diff
@@ -3,12 +3,13 @@
   src=""
   class="video-js vjs-default-skin"
   controls
-  width="829" height="511"
+  height="560"
   data-setup='{{ data_setup }}'
 >
   {% for transcript in player_state.transcripts %}
     <track kind="captions" src="{{ transcript.url }}" srclang="{{ transcript.lang }}" label="{{ transcript.label }}"
     {% if forloop.first %}default{% endif %}>
   {% endfor %}
 </video>
+<!-- height="560" it's height of iframe minus height of control bar -->
```

Comment:
How will this work on different parent block widths?

Also please move this comment closer to the thing you're commenting.